docs/Homebrew-Governance.md

```diff
@@ -48,13 +46,11 @@
 
 4. Any approved amendments will take effect three weeks after the close of voting.
 
-5. No amendment may be proposed which violates Software Freedom Conservancy policies. The PLC may veto such an amendment by an ordinary resolution.
-
 ## 4. Project Leadership Committee
 
-1. The governance and operation of Homebrew is determined by the PLC. The PLC will represent Homebrew in all dealings with the Software Freedom Conservancy (SFC). The PLC has final authority in all matters related to the operation of Homebrew, and will comply with all Software Freedom Conservancy policies.
+1. The financial administration of Homebrew, organisation of the AGM, enforcement of the code of conduct and removal of members are performed by the PLC. The PLC will represent Homebrew in all dealings with OpenCollective.
 
-2. The PLC consists of five members. Committee members are elected by Homebrew members in a [Meek Single Transferable Vote](https://en.wikipedia.org/wiki/Counting_single_transferable_votes#Meek) election using the Droop quota. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
+2. The PLC consists of five members including the Project Leader. Committee members are elected by majority vote of Homebrew members. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
```

Comment:
Apologies for missing the private discussion, but I just wanted to clarify because my understanding was that the change was necessary only due to the vacant spot this cycle. In the future, if there is more interest, will we go back to the PLC being five voting members and having the PL be a sixth, non-voting member? Or will it forever be four voting and 1 non-voting?

Reply:
Can we decide on that next cycle? Or is it important for this one to decide on the future (2023 onwards) makeup of the PLC? The point of this change is an amendment to make sure that we aren't contravening our own governance docs this time around. This was due to an oversight of mine when finding candidates for the vacant PLC positions this year.

Reply:
> Apologies for missing the private discussion, but I just wanted to clarify because my understanding was that the change was necessary only due to the vacant spot this cycle.

That's the most urgent short-term issue. Longer-term: I don't think it's really worked having the PL not be on the PLC. It creates information silos and in the last year many of the PLC responsibilities have been shared or nudged to start by the PL.

I'm definitely open to having the governance documents be more "open" i.e. prepare for a world where the PL/PLC/TSC are all entirely different groups from today and instead make it e.g. optional for the PL to be on PLC/TSC, etc.

Reply:
Sorry for the delay. Totally fine with me if we decide later, I was just curious what the long-term plan was.